website/docs/guides/core-to-cloud-2.md

```diff
@@ -0,0 +1,196 @@
+hoverSnippet: "Use this guide to understand the considerations and methods you need to move from dbt Core to dbt Cloud."
+icon: 'guides'
+hide_table_of_contents: true
+tags: ['Migration','dbt Core','dbt Cloud']
+keywords: ['dbt Core','dbt Cloud','Migration', 'Move dbt', 'Migrate dbt']
+level: 'Intermediate'
+recently_updated: true
+---
+
+## Introduction
+Moving from dbt Core to dbt Cloud streamlines analytics engineering workflows by allowing teams to develop, test, deploy, and explore data products using a single, fully managed platform.
+
+Explore our 3-part-guide series on moving from dbt Core to dbt Cloud. The series is ideal for users aiming for streamlined workflows and enhanced analytics:
+
+import CoretoCloudTable from '/snippets/_core-to-cloud-guide-table.md';
+
+<CoretoCloudTable/>
+
+## What you'll learn
+Migrating from dbt Core to dbt Cloud can yield significant benefits and a standardized way of working. Understanding the differences between dbt Cloud and your current Core deployment will help you strategize and plan for your move.
+
+The guide outlines the following steps:
+
+- [Considerations](https://docs.getdbt.com/guides/core-to-cloud-2?step=3): Learn about the most important things you need to think about when moving from Core to Cloud.
+- [Plan your move](https://docs.getdbt.com/guides/core-to-cloud-2?step=4): Considerations you need to make, such as user roles and permissions, onboarding order, current workflows, and more.
+- [Move to dbt Cloud](https://docs.getdbt.com/guides/core-to-cloud-2?step=5): Review the steps to move your dbt Core project to dbt Cloud, including setting up your account, data platform, and Git repository.
+- [Test and validate](https://docs.getdbt.com/guides/core-to-cloud-2?step=6): Discover how to ensure model accuracy and performance post-move,.
+- [Transition and training](https://docs.getdbt.com/guides/core-to-cloud-2?step=7): Learn how to fully transition to dbt Cloud and what training and support you may need. 
+- [Summary](https://docs.getdbt.com//guides/core-to-cloud-2?step=8): Summarizes key takeaways and what you've learned in this guide.
+- [What's next?](https://docs.getdbt.com/guides/core-to-cloud-2?step=9): Introduces what to expect in the following guides.
+
+## Considerations
+
+If your team has a custom deployment of dbt Core today, you could be reading this guide because:
+- You’ve realized the burden of maintaining that deployment.
+- The person who set it up has since left.
+- Or you’re interested in what dbt Cloud could do to standardize your deployment and consolidate some of the toolings that you may already have in place.
+
+This guide shares the technical adjustments and team collaboration strategies you’ll need to know to move your project from dbt Core to dbt Cloud.  Each “BYO” (build your own) deployment of dbt Core will look a little different, but after seeing hundreds of teams make the migration, there are many things in common.
```

Comment:
BYO has a canonically different meaning and it might be something to avoid
```suggestion
This guide shares the technical adjustments and team collaboration strategies you’ll need to know to move your project from dbt Core to dbt Cloud.  Each "build your own" deployment of dbt Core will look a little different, but after seeing hundreds of teams make the migration, there are many things in common.
```